docs/docs/configuration.md

```diff
@@ -204,6 +204,23 @@ For example:
 # In this example, the following will not be packaged in to MTA archive:
 # all files with the"txt" extension and the "mtaignore" folder within the "myfolder" directory. 
      
+```
+From MBT 1.2.25 version, the `build-parameters ignore` attribute will be upgraded to support [full glob pattern](https://en.wikipedia.org/wiki/Glob_(programming)). Wildcards, such as `*`,`!`,`?` can be used to match characters. 
+
+By using negation pattern `!` in the `build-parameters ignore` attribute, because `not ignore` equal to `include`, it means you can select specified content into the package.
```

Comment:
By using the `!` negation pattern in the `build-parameters ignore` attribute, you can select specified content into the package (because `not ignore` equals `include`).